components/notebook-controller/config/manager/manager.yaml

```diff
@@ -10,6 +10,10 @@ kind: Deployment
 metadata:
   name: deployment
 spec:
+  strategy:
```

Comment:
That `spec.strategy`  option is already included on the managers' deployment files in both branches `main` and `v1.9-branch` 

https://github.com/opendatahub-io/kubeflow/blob/main/components/notebook-controller/config/manager/manager.yaml#L13-L16